cumulus_library/studies/core/count_core.py

```diff
@@ -58,29 +58,11 @@ def _count_encounter_type(view_name, cols, duration):
 
 def count_encounter_type(duration=None):
     cols = [
-        "enc_class_display",
-        "enc_type_display",
-        "enc_service_display",
-        "enc_priority_display",
+        "codableconcept_display",
```

Comment:
You frequently use the actual English word `codable` vs the (afaict) not-a-word `codeable` from the spec (codable is in Merriam Webster, but it still gets squiggle marks in my browser, so maybe they are both barely words). That's fine, just calling it out in case you were fat-fingering these cases. But I'd also defend your right to use a maybe-more-correct word in code and only use the spec word when you have to.

Reply:
i actually think i should use the spec name here, despite what my spellchecker is telling me to do otherwise, because an analytics user would look for it in the non-english way.

fun fact, since i am the kind of person to go look this sort of thing up for fun: while Miriam Webster does not thing 'codeable' is a word, the OED does recognize it as valid, so maybe this is a color/colour situation. Anyway, taking a lorry back to my flat.

Reply:
Merriam Webster is my household's (of one, granted) official dictionary for Boggle. So, I don't know why anyone would bother looking at another authority.